catalog/core/catalog-core-standardframework/src/test/java/ddf/catalog/cache/solr/impl/SortedQueryMonitorTest.java

```diff
@@ -215,6 +228,106 @@ public void interruptThirdFuture() throws Exception {
             NullPointerException.class, InterruptedException.class, InterruptedException.class);
   }
 
+  @Test
+  public void testSorts() throws Exception {
```

Comment:
✏️  Should add tests with null attribute found in the result and with non-`Comparable` values.

Reply:
Fixed